airbyte-integrations/connectors/source-mongodb-internal-poc/src/main/java/io/airbyte/integrations/source/mongodb/internal/MongoConstants.java

```diff
@@ -5,16 +5,26 @@
 package io.airbyte.integrations.source.mongodb.internal;
 
 import io.airbyte.integrations.debezium.internals.mongodb.MongoDbDebeziumConstants;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
 
 public class MongoConstants {
 
   public static final String AUTH_SOURCE_CONFIGURATION_KEY = MongoDbDebeziumConstants.Configuration.AUTH_SOURCE_CONFIGURATION_KEY;
   public static final Integer CHECKPOINT_INTERVAL = 1000;
```

Comment:
The default checkpoint interval in other databases is 10K records so lets keep it 10K as well

Reply:
Will fix.